### DISCUSSION TITLE
665p, RCTEventEmitter depracated 관련 문의드립니다

### DISCUSSION THREAD

**Discussion by rudenick0309** [velopert/dealing-with-react-native#21] [Q&A] [answered]
665페이지, 
11.1.9.1 이벤트 발생시키기 파트에서, 
RCTEventEmitter 가 depracated 되서 함수가 먹히지를 않습니다.
빌드는 성공적으로 돌아가구요
해당 부분은 무엇으로 교체해야 정상적으로 돌아가나요?

**Comment by velopert** [comment, marked as answer]
RCTEventEmitter가 RN에서 deprecated되진 않았고, 네이티브쪽 구현에 있어서 deprecated된 코드가 사용됐었나 봅니다.

과거에 잠깐 발성했었고 RN을 업데이트하면 해결 된다고 해요.
